daprdocs/content/en/operations/monitoring/newrelic.md

```diff
@@ -0,0 +1,347 @@
+
+![New Relic Kubernetes Automated Installer](/images/nr-automated-installer.png)
+
+## Deploy integration
+
+To deploy, run this command in your console, and insert the path to where you downloaded the manifest.
+
+```bash
+kubectl apply -f newrelic-manifest.yaml -n dapr-monitoring
+```
+
+## Validation
+
+### Ensure New Relic Kubernetes integration is running.
+
+```bash
+kubectl get pods -n dapr-monitoring
+NAME                                            READY STATUS  RESTARTS  AGE
+nri-bundle-kube-state-metrics-f7cf766ff-2bdt6   1/1   Running 0         16m
+nri-bundle-newrelic-infrastructure-48wdk        1/1   Running 0         16m
+nri-bundle-newrelic-infrastructure-fj4pz        1/1   Running 0         16m
+nri-bundle-newrelic-logging-7gw8k               1/1   Running 0         16m
+nri-bundle-newrelic-logging-b6fwx               1/1   Running 0         16m
+nri-bundle-nri-kube-events-6899758c57-rfmvq     2/2   Running 0         16m
+nri-bundle-nri-prometheus-689d7b6f45-gfrgx      1/1   Running 0         16m
+```
+
+### Ensure Dapr is running
+
+```bash
+kubectl get pods -n dapr-system
+NAME                                READY STATUS  RESTARTS  AGE
+dapr-operator-59ddb59f4c-4fgzm      1/1   Running 0         16m
+dapr-placement-599d87d5cb-n6zpc     1/1   Running 0         16m
+dapr-sentry-777658c665-z4qhb        1/1   Running 0         16m
+dapr-sidecar-injector-56c4588-4rs8z 1/1   Running 0         16m
+```
+
+## New Relic Kubernetes Cluster Explorer
+
+The [New Relic Kubernetes Cluster Explorer](https://docs.newrelic.com/docs/integrations/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer) provides a unique visualization of the entire data and deployments of the data collected by the Kubernetes integration.
+
+![New Relic Kubernetes Cluster Explorer](/images/nr-k8s-cluster-explorer.png)
+
+## Install Quickstart application Distributed calculator
+
+### Basics
+
+This quickstart shows method invocation and state persistent capabilities of [Dapr through a distributed calculator](https://github.com/dapr/quickstarts/tree/master/distributed-calculator) where each operation is powered by a different service written in a different language/framework:
+
+Addition: Go [mux](https://github.com/gorilla/mux) application
+Multiplication: Python [flask](https://flask.palletsprojects.com/en/1.0.x/) application
+Division: Node [Express](https://expressjs.com/) application
+Subtraction: [.NET Core](https://docs.microsoft.com/en-us/dotnet/core/) application
+The front-end application consists of a server and a client written in [React](https://reactjs.org/).
+
+The below illustration shows the underlying architecture:
+
+![Dapr Distributed Calculator Architecture](/images/nr-dapr-distributed-calculator.png)
+
+### State Store
+
+For state store I am using Redis and deploy it using the following command:
```

Comment:
Again, the use of Redis is not required to enable New Relic.
(Also note, we do not use first person in the docs)